server/cmd/bblfsh-web/main.go

```diff
@@ -33,8 +41,14 @@ func main() {
 		return
 	}
 
+	if level, ok := logLevels[os.Getenv("LOG_LEVEL")]; ok {
+		logrus.SetLevel(level)
+	}
+
 	if !debug {
 		gin.SetMode(gin.ReleaseMode)
+	} else {
+		logrus.SetLevel(logrus.TraceLevel)
```

Comment:
Isn't it a bit confusing that `--debug` make logging at `TraceLevel`? Why not just make `--debug` the same as passing `--log-level=debug`?

Reply:
`--debug` mode in `gin` is very verbose, so I think we could do the same for the app logs.
But yes, we can also configure this independently, so setting `debug` mode in `gin`, and app logging level will be independent.

Reply:
I don't know `gin`, but yup. I would simply do nothing for `logrus` depending on the `--debug` level. So we would use `--debug` to change `gin` mode, and `--log-level` to set the app log level. If you could also mention this in the description of the `--debug` flag.